packages/schema-dot-org-types/tools/transform-schema.js

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
```

Comment:
Yep! All the benefits of encapsulation and sibling dependency, none of the public drawbacks. :)